handbook/people-ops/from-graphbook/holidays.md

```diff
@@ -1,17 +1,15 @@
-# Holiday schedule
+# Holidays
 
-The Company’s office is closed on the following dates: 
+We have a team that is based throughout many parts of the world and we respect everyone's local holidays. Therefore, there is no need for teammates to provide a specific enumeration of holidays by country, as holidays are explicitly covered as part of our unlimited PTO policy. 
```

Comment:
Can you link the phrase "unlimited PTO policy" to wherever we talk about it in the handbook (if we do)?

Reply:
Yes– just submitted a PR to add our vacation/FTO policy! Will link to it once approved.